src/engraving/libmscore/ledgerline.cpp

```diff
@@ -46,6 +46,10 @@ LedgerLine::LedgerLine(Score* s)
     _next       = 0;
 }
 
+LedgerLine::~LedgerLine()
```

Comment:
are empty destructors recommended in the style guide?

Reply:
no, the code style guide doesn't formalize it.
I added an empty destructor to put a breakpoint and see where these objects are being deleted, to see why so many of them are created and deleted during layout
https://github.com/musescore/MuseScore/pull/9100